source/examples/Migrations/LocalMigration/LocalMigration.java

```diff
@@ -1,22 +1,42 @@
 public class Migration implements RealmMigration {
   @Override
   public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
+     Long version = oldVersion;
 
      // DynamicRealm exposes an editable schema
      RealmSchema schema = realm.getSchema();
-     
-     if (oldVersion == 1) {
-        schema.create("Person")
-            .addField("fullName", String.class)
-            .addField("age", int.class);
-        
-        realm.beginTransaction();
-        for (Person person : realm.where(Person.class).findAll()) {
-            person["fullName"] = person["firstName"] + " " + person["lastName"];
-        }
-        realm.commitTransaction();
 
-        oldVersion++;
+     // Changes from version 0 to 1: Adding lastName.
+     // All properties will be initialized with the default value "".
+     if (version == 0L) {
+        schema.get("Person")
+            .addField("lastName", String.class, FieldAttribute.REQUIRED);
+        version++;
+     }
+
+     // Changes from version 1 to 2: combine firstName/lastName into fullName
+     if (version == 1L) {
+        schema.get("Person")!!
```

Comment:
```suggestion
        schema.get("Person")
```
`!!` is only supported in Kotlin